src/types/shared.ts

```diff
@@ -34,3 +34,10 @@ export interface FluxToolbarFunction {
 }
 
 export type ErrorMessageComponent = ComponentType<{error: Error}>
+
+export interface Pageable {
+  currentPage: number
+  rowsPerPage: number
+  totalPages: number
+  paginate: (newPage: number) => void
+}
```

Comment:
moved out of https://github.com/influxdata/ui/blob/master/src/tasks/pagination/TasksList.tsx#L63